x/ibc/testing/chain.go

```diff
@@ -567,7 +567,7 @@ func (chain *TestChain) CreateTMClientHeader(chainID string, blockHeight int64,
 		AppHash:            chain.CurrentHeader.AppHash,
 		LastResultsHash:    tmhash.Sum([]byte("last_results_hash")),
 		EvidenceHash:       tmhash.Sum([]byte("evidence_hash")),
-		ProposerAddress:    tmValSet.Proposer.Address, //nolint:staticcheck
+		ProposerAddress:    tmValSet.Proposer.Address,
```

Comment:
Please revert this line:

```suggestion
		ProposerAddress:    tmValSet.Proposer.Address, //nolint:staticcheck
```